tests/structure_test_tests.sh

```diff
@@ -22,14 +22,15 @@
 
 failures=0
 # build newest structure test binary
-go test -c github.com/GoogleCloudPlatform/container-structure-test -o structure-test
+go build -o structure-test .
```

Comment:
should this be using the makefile rule so it doesnt get out of sync? I think it needs ldflags and tags?

Reply:
yeah it definitely should